src/app/characters/character-list/character-list.component.spec.ts

```diff
@@ -26,16 +29,19 @@ class ComicListStubComponent {
   @Input() character;
 }
 
-const fakeCharacters = [{
-  id: 1,
-  name: 'Fake character',
-  description: 'My fake super hero',
-  thumbnail: {
-    path: 'Fake path',
-    extension: 'fake'
-  },
-  urls: [{ url: 'http://fakeurl/', type: 'fakeType' }]
-}] as Character[];
+const fakeMarvelResponseData = {
+  results: [{
+    id: 1,
+    name: 'Fake character',
+    description: 'My fake super hero',
+    thumbnail: {
+      path: 'Fake path',
+      extension: 'fake'
+    },
+    urls: [{ url: 'http://fakeurl/', type: 'fakeType' }]
+  }] as Character[],
+  total: 1
+} as MarvelResponseData<Character>;
```

Comment:
I think that we can directly define the type of the `fakeMarvelResponseData` variable instead of type casting it.

Reply:
Done.